.github/DocsPostProcessor/Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.5
```

Comment:
This is going to be too new for the pipeline.

You could update it to:
```suggestion
// swift-tools-version:5.4
```
and that should work.  Assuming Soup doesn't also require 5.5